src/main/java/org/llorllale/cactoos/matchers/EndsWith.java

```diff
@@ -53,7 +53,8 @@ public EndsWith(final Text text) {
             new TextMatcher(
                 new MatcherOf<>(
                     (String act) -> act.endsWith(text.asString()),
-                    text
+                        desc -> desc.appendText(text.asString()),
+                        (actual, desc) -> desc.appendValue(actual)
```

Comment:
@ruca905 can you add a new todo so that we add a constructor to `TextMatcher` taking a `Func<String, Boolean>` in order to simplify this and all the other `TextMatcher`?